TaylorFramework/Modules/caprices/ProccessingMessage/PathFormatter.swift

```diff
@@ -20,56 +20,56 @@ extension Path {
     
     func lastComponentFromPath() -> String {
         if self.isEmpty { return String.Empty }
-        let pathComponents = self.componentsSeparatedByString(FilePath.Separator)
+        let pathComponents = self.components(separatedBy: FilePath.Separator)
         return pathComponents.last! // Safe to force unwrap
     }
     
     
-    func absolutePath(analyzePath: String = NSFileManager.defaultManager().currentDirectoryPath) -> Path {
-        if self.hasPrefix(TildeSymbol) { return NSHomeDirectory() + self.stringByReplacingOccurrencesOfString(TildeSymbol, withString: String.Empty) }
+    func absolutePath(_ analyzePath: String = FileManager.default.currentDirectoryPath) -> Path {
+        if self.hasPrefix(TildeSymbol) { return NSHomeDirectory() + self.replacingOccurrences(of: TildeSymbol, with: String.Empty) }
```

Comment:
Done, used "" instead of String.Empty (simple extension on String where Empty = "") in fact that it's faster